set.mm

```diff
@@ -88360,7 +88380,7 @@ the first case of his notation (simple exponentiation) and subscript it
     wbr ) ABDFZCAGZCEHZICDCJZTUACUCDCKZLZUBCUCMSZABCDNUBUEUFCUCEUDOPQR $.
 
   $( A one-to-one function's image under a subset of its domain is equinumerous
```

Comment:
```suggestion
  $( The image under a one-to-one function of a subset of its domain is equinumerous
```

Reply:
I'm holding off on making this change because I find this wording harder to parse than the original phrasing in set.mm. I'd like to see some other opinions on this before making a decision though.

Reply:
The problem is that the original phrasing is incorrect: you don't say "the image of F under A" but "the image of A under F" (when F is a function and A is a subset of its domain). I know the original phrasing is not yours, but it's a good occasion to correct it.

Reply:
Maybe something like "if a function is one-to-one, then the image of a subset of its domain under itself" would work?

Reply:
This is also correct (except I would write "under it" instead of "under itself").  I think "by" is also an acceptable preposition for the function, so it could be "The image by a one-to-one function of a subset of its domain..." All three forms are fine with me. (I prefer my original formulation, but note that my mother tongue is not English.)